ribs/visualize.py

```diff
@@ -453,3 +453,145 @@ def sliding_boundaries_archive_heatmap(archive,
 
     # Create the colorbar.
     ax.figure.colorbar(t, ax=ax, pad=0.1)
+
+
+def parallel_axes_plot(archive,
+                       ax=None,
+                       bc_order=None,
+                       axis_labels=None,
+                       cmap="magma",
+                       linewidth=1.5,
+                       alpha=0.8,
+                       vmin=None,
+                       vmax=None):
+    """Plots a parallel axes plot of an archive with an N-Dimensional behavior
+    space.
+
+    This visualization is meant to see the coverage of the behavior space at a
+    glance. It is possible to glean the correlations between consecutive axes,
+    but the results should be interpreted carefully.
+
+    Examples:
+        .. plot::
+            :context: close-figs
+
+            >>> import numpy as np
+            >>> import matplotlib.pyplot as plt
+            >>> from ribs.archives import GridArchive
+            >>> from ribs.visualize import grid_archive_heatmap
+            >>> # Populate the archive with the negative sphere function.
+            >>> archive = GridArchive([20, 20, 20], [(-1, 1), (-1, 1), (-1, 1)])
+            >>> archive.initialize(solution_dim=2)
+            >>> for x in np.linspace(-1, 1, 100):
+            ...     for y in np.linspace(-1, 1, 100):
+            ...         for z in np.linspace(-1, 1, 100):
+            ...             archive.add(solution=np.array([x,y,z]),
+            ...                         objective_value=-(x**2 + y**2 + z**2),
+            ...                         behavior_values=np.array([x,y,z]))
+            >>> # Plot a heatmap of the archive.
+            >>> plt.figure(figsize=(8, 6))
+            >>> parallel_axes_plot(archive)
+            >>> plt.title("Negative sphere function")
+            >>> plt.ylabel("axis values")
+            >>> plt.show()
+
+    Args:
+        archive (SlidingBoundariesArchive): A 2D SlidingBoundariesArchive.
+        ax (matplotlib.axes.Axes): Axes on which to plot the heatmap. If None,
+            the current axis will be used.
+        bc_order (list<str>): By default, the BCs of the parallel axes will
+            appear in-order. To switch the axes around, supply a list of the
+            behavior axes (e.g., ['behavior_1', 'behavior_0', 'behavior_2']).
```

Comment:
The list thing can be written so it shows up as code. The docstrings are formatted in restructured text, so use this to show "abcde" as `abcde`:
```
``abcde``
```